### ISSUE TITLE
Adding R3F code examples to docs 

### ISSUE THREAD

**Issue by austinpapritz** [pmndrs/maath#41]
I'm planning on going through the R3F code examples for uses of maath and adding them to the docs here.

**Comment by austinpapritz** [comment]
[maath-code-samples.txt](https://github.com/user-attachments/files/15847754/maath-code-samples.txt)

**Comment by suggoooiii** [comment]
you have a pretty good list here, maybe we can stat by listing the functions that don't have examples

**Comment by austinpapritz** [comment]
> you have a pretty good list here, maybe we can stat by listing the functions that don't have examples

Thanks! Great suggestion, I updated the list with what is missing.

**Comment by austinpapritz** [comment]
> let's try and make our own examples for everything that's missing, i think that would be amazing. im down if you are 👀

I might have time to help out. What did you have in mind?

**Comment by suggoooiii** [comment]
yeah i don't understand some of the stuff either but we'll figure it out at the end i'm pretty sure.
Yeah i was actually thinking of making smaller projects for each missing eample through codesandbox

**Comment by austinpapritz** [comment]
Maybe easier than a whole project, creating a simple graphic like this for the easing functions might better show the differences.
https://www.youtube.com/shorts/I9E2H83FnGs
